proposals/lip-0063.md

```diff
@@ -608,7 +608,7 @@ def createGenesisDataObj():
     genesisDataObj = {}
     genesisDataObj.initRounds = POS_INIT_ROUNDS
     roundLength = NUMBER_ACTIVE_VALIDATORS + NUMBER_STANDBY_VALIDATORS
-    r = ceiling(a.header.height - HEIGHT_PREVIOUS_SNAPSHOT_BLOCK, roundLength)
+    r = ceiling(a.header.height, roundLength)
```

Comment:
since the round number computation is not consistent within the SDK versions, and the one from SDK 5 is also a bit counter-intuitive (because of the change of the round length in the past), I would suggest to describe it verbally as we had it in the past:
"r = round number for the block a"

Reply:
I have changed the function as Jan suggested.